events/Random Events.txt

```diff
@@ -825,6 +810,7 @@ country_event = {
 	option = {
 		name = "Put together a naval force to seek out and quell this monster"
 		money = -500
+		plurality = -0.2
```

Comment:
Have you verified that decimal values work in the effect? We should add some plurality if the realm decides to ignore it

Reply:
It gives 1 plurality, when ignored. The decimal value for plurality seems to work as intended, although it only is shown on the next day or when other effects update the shown value, like triggering the event multiple times on the same day via decision.